### PR TITLE
blog: adds saga pattern blog

### PR DESCRIPTION
**Pull request by coltmcnealy-lh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <a href="https://littlehorse.io/"><img alt="littlehorse.io" src="https://raw.githubusercontent.com/littlehorse-enterprises/littlehorse/01ef9618b7b719c3aac1d2e2eb5096f164a659b1/docs/images/badge-littlehorse-io.svg"/></a>
 <a href="https://littlehorse.dev/"><img alt="littlehorse.dev" src="https://raw.githubusercontent.com/littlehorse-enterprises/littlehorse/01ef9618b7b719c3aac1d2e2eb5096f164a659b1/docs/images/badge-littlehorse-dev.svg"/></a>
-[![slack](https://img.shields.io/badge/Slack-511651.svg?logo=slack)](https://kubernetes.slack.com/messages/C87MF1RFD)
+[![slack](https://img.shields.io/badge/Slack-511651.svg?logo=slack)](https://launchpass.com/littlehorsecommunity)
```
Comment (coltmcnealy-lh) [inline README.md:5]: No idea where this came from, but it definitely didn't point to our slack. We aren't on `kubernetes.slack.com`

File: docs/blog/2023-09-01-helm-and-k8s-operators.md
Comment (coltmcnealy-lh) [inline docs/blog/2023-09-01-helm-and-k8s-operators.md:1]: Moved to `docs.littlehorse.io` as this has nothing to do with LH

File: docs/blog/2024-09-04-basics-of-workflows.md
Comment (coltmcnealy-lh) [inline docs/blog/2024-09-04-basics-of-workflows.md:1]: Moved it over and prettified it

File: README.md (hunk 1 above)
Comment (Snarr) [reply README.md:5]: Great catch, that was probably a typo on my end from adding the Shields but I have no idea where I got a link to the Kubernetes slack.

File: docs/blog/2024-09-24-saga-pattern.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+At a technical level, the [SAGA Pattern](https://microservices.io/patterns/data/saga.html) allows you to perform distributed transactions across multiple systems (microservices, API's, mainframes, etc) without 2-phase commit.
+
+In plain English, it is a tool in the belt of a software engineer to prevent half-fulfilled bank transfers, hanging orders, or other failures which would result in a Grumpy Customer.
+
+### Use Cases
+
+Business processes often need to perform actions in two separate systems either all at once or not at all. For example, you may need to charge a customer's credit card, reserve inventory, and ship an item to the customer all at once or not at all. If the payment went through but shipping failed, we would see the Grumpy Customer Problem yet again.
+
+The SAGA pattern is appropriate when:
+* A business process must take action across multiple separate systems (legacy monoliths, microservices, external API's, etc),
```
Comment (mitchell-h) [inline docs/blog/2024-09-24-saga-pattern.md:35]: Remove the ()'s.  They don't add anything to this point.

**Review by Snarr — APPROVED** [review]
